srv/modules/runners/disk_led.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+"""
+This runner is used to turn on/off the ident and fault LED of a disk.
+"""
+from __future__ import absolute_import
+import logging
+import re
+import salt.client
+
+
+log = logging.getLogger(__name__)
+
+
+def _process(hostname, device_name, led, status):
+    """
+    Turn on/off the specified LED of the disk on the given host.
+    """
+    local_client = salt.client.LocalClient()
+    device_file = '/dev/{}'.format(device_name)
+    status = 'on' if status in [True, 'on'] else 'off'
+
+    # Get the configuration of the command to turn on/off the ident
+    # or fault LED on the target host.
+    cmd_cfg = local_client.cmd(
+        hostname, 'pillar.get', ['disk_led:cmd'],
+        tgt_type='compound')[hostname]
+
+    print("Turning {} the {} LED of disk '{}' on host '{}' ...".format(
+        status, led, device_file, hostname))
+    _cmd_run(hostname, cmd_cfg[led][status].format(device_file=device_file))
+
+
+def _cmd_run(hostname, cmd):
+    """
+    Run the specified command line.
+    .. note:: This function has been introduced for easier unit testing.
+    """
+    local_client = salt.client.LocalClient()
+    local_client.cmd(hostname, 'cmd.run', [cmd], tgt_type='compound')
+
+
+def osd(osd_id, device_type, led, status):
+    """
+    Manage the storage enclosure LED by OSD identifier.
+    :param osd_id: The OSD id, e.g.
+        - 4
+        - osd.1
+    :type osd_id: str
+
+    :param device_type: The device to use. This can be:
+        - primary
+        - journal
+        - db
```

Comment:
Bluestore can also be deployed on 3 devices: primary, db and wal

Reply:
Don't understand the intention of your comment. The parameters of this function are derived from the orchestrator interface:
```
ceph orchestrator osd {ident,fault}-{on,off} {primary,journal,db,all} <osd-id>
```

Reply:
Then that interface is incomplete. Bluestore can potentially use 3 devices. Maybe it was left out intentionally as those deployments are indeed rare? @sebastian-philipp any thoughts?

Reply:
@liewegas Any comments on that?

Reply:
So, the question is: `journal` the same as `wal` or is `journal` the same as `db`?

Reply:
from #ceph-devel
 [18:46:57] <sageyphone> The original idea was journal was the filestore journal.
[18:47:13] <sageyphone> No equivalence between it and db or wal

Reply:
Issue is addressed now